### ISSUE TITLE
Correct nesting of #global-masthead inside #local-masthead

### ISSUE THREAD

**Issue by khawkins98** [ebiwd/EBI-Framework#74]
In v1.1 we improved the structure of the masthead bars. 

An inadvertent consequence was that `div#global-masthead` was nested inside `#local-masthead` -- a confusing structure (note we also have a `.masthead`):

``` html
<div id="local-masthead">
  <header>
    <div id="global-masthead" class="clearfix">
      <!-- EMBL-EBI Black bar content -->
    </div>
    <div class="masthead row">
      <!-- Local service name, menu -->
    </div>
  </header>
</div>
```

**Comment by khawkins98** [comment]
Here's the fix we're working in v1.2 that maintains the improved structure and looks to improve semantics:
- `#local-masthead` becomes  `#masthead` (and we add a `.masthead` for themeing)
- `#global-mathead` becomes `#masthead-black-bar`  (and we add a `.masthead-black-bar` for themeing)
- `.masthead` becomes `.masthead-inner`

NB: we'll keep the IDs for `script.js`

**Comment by khawkins98** [comment]
This should now be complete.
